server/backend/src/users/users.controller.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Controller,
+  ForbiddenException,
+  Get,
+  Inject,
+  Param,
+} from '@nestjs/common';
+import { NotFoundError } from '../errors.types';
+import { UsersService } from './users.service';
+
+export type GetGroupsOutput = {
+  id: string;
+  name: string;
+  cameras: {
+    id: string;
+    name: string;
+  }[];
+}[];
+
+@Controller('users')
+export class UsersController {
+  constructor(@Inject(UsersService) private usersService: UsersService) {}
+
+  @Get(':id/groups')
+  async getGroups(@Param('id') id: string): Promise<GetGroupsOutput> {
+    try {
+      const groupsJSON = (await this.usersService.getGroups(id)).toJSON();
+      return groupsJSON.map((group) => ({
+        id: group.id,
```

Comment:
discussed offline: please make sure to document json format of the endpointsssss output smiley face :)

Reply:
Will do in our design doc :)))